ethereum/contracts/mock/MockRelayerIntegration.sol

```diff
@@ -86,20 +88,20 @@ contract MockRelayerIntegration is IWormholeReceiver {
 
     function vaaKeysCreator(uint64 sequence1, uint64 sequence2)
         internal view
-        returns (IWormholeRelayer.VaaKey[] memory vaaKeys)
+        returns (VaaKey[] memory vaaKeys)
     {
-        vaaKeys = new IWormholeRelayer.VaaKey[](2);
-        vaaKeys[0] = IWormholeRelayer.VaaKey(
-            IWormholeRelayer.VaaKeyType.EMITTER_SEQUENCE,
+        vaaKeys = new VaaKey[](2);
+        vaaKeys[0] = VaaKey(
+            VaaKeyType.EMITTER_SEQUENCE,
             wormhole.chainId(),
-            relayer.toWormholeFormat(address(this)),
+            toWormholeFormat(address(this)),
             sequence1,
             bytes32(0x0)
         );
```

Comment:
Hm, yeah, in this case I'd use the object notation because `bytes32(0x0)` doesn't tell me anything :sweat_smile:
When several parameters have the same type, I think it might be worth it. Even with good variable naming the compiler won't catch some argument swaps. Only tests would.